types/react-redux/index.d.ts

```diff
@@ -164,7 +164,7 @@ interface MergeProps<TStateProps, TDispatchProps, TOwnProps, TMergedProps> {
     (stateProps: TStateProps, dispatchProps: TDispatchProps, ownProps: TOwnProps): TMergedProps;
 }
 
-interface Options extends ConnectOptions {
+interface Options<TStateProps = any, TOwnProps = any, TMergedProps = any> extends ConnectOptions {
```

Comment:
Why default to `any`? I would argue that this is dangerous and can give people a feeling of safety when in actuality `any` has been inferred without their knowledge.

My preference is not to default this at all, and if it cannot be inferred, it will be inferred to `{}` which will force people to deal with the issue.

If you want `any` you should have to declare that explicitly.